<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>240</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../apprcc.qrc">
    <normaloff>:/img/icon/熊猫.png</normaloff>:/img/icon/熊猫.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	font-size: 16px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-weight: bold;
 	border-radius:9px;
	background:rgba(66, 195, 255, 0);
	color: rgb(218, 218, 218);
}
QDialog{
background-color: rgba(75, 75, 75, 120);
border: 0px solid #42adff;
border-radius:0px;}
QPushButton {
	font-size: 18px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-weight: bold;
 	border-radius:9px;
	background:rgba(66, 195, 255, 0);
	color: rgb(218, 218, 218);
}
QPushButton:focus{
	outline: none;
}
QPushButton::pressed{
	font-family: &quot;Microsoft YaHei&quot;;
    font-size: 14px;
    font-weight: bold;
    color:rgb(200,200,200);
    text-align: center center;
    padding-left: 5px;
	padding-right: 5px;
	padding-top: 4px;
	padding-bottom: 4px;
	border-style: solid;
	border-width: 0px;
	border-color: rgba(255, 255, 255, 255);	
	border-radius: 3px;
	background-color:  #bf513b;
}
QPushButton::hover {
	border-style: solid;
	border-width: 0px;
	border-radius: 0px;
	background-color: rgba(48,148,243,80);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,0,0">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <property name="bottomMargin">
      <number>11</number>
     </property>
     <item row="4" column="0">
      <widget class="QLabel" name="letterboxImageLabel">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>letterboxImage</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="maxBoxesLabel">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>最大框数：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="modelButton">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>file</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{font-family: &quot;Microsoft YaHei&quot;;
font-size: 14px;
font-weight: bold;
color:white;
text-align: center center;
padding-left: 5px;
padding-right: 5px;
padding-top: 4px;
padding-bottom: 4px;
border-style: solid;
border-width: 0px;
border-color: rgba(255, 255, 255, 255);
border-radius: 3px;
background-color: rgba(200, 200, 200,0);}

QPushButton:focus{outline: none;}

QPushButton::pressed{font-family: &quot;Microsoft YaHei&quot;;
                     font-size: 14px;
                     font-weight: bold;
                     color:rgb(200,200,200);
                     text-align: center center;
                     padding-left: 5px;
                     padding-right: 5px;
                     padding-top: 4px;
                     padding-bottom: 4px;
                     border-style: solid;
                     border-width: 0px;
                     border-color: rgba(255, 255, 255, 255);
                     border-radius: 3px;
                     background-color:  #bf513b;}

QPushButton::disabled{font-family: &quot;Microsoft YaHei&quot;;
                     font-size: 14px;
                     font-weight: bold;
                     color:rgb(200,200,200);
                     text-align: center center;
                     padding-left: 5px;
                     padding-right: 5px;
                     padding-top: 4px;
                     padding-bottom: 4px;
                     border-style: solid;
                     border-width: 0px;
                     border-color: rgba(255, 255, 255, 255);
                     border-radius: 3px;
                     background-color:  #bf513b;}
QPushButton::hover {
border-style: solid;
border-width: 0px;
border-radius: 0px;
background-color: rgba(48,148,243,80);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../apprcc.qrc">
         <normaloff>:/img/icon/打开.png</normaloff>:/img/icon/打开.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nms_iouLael">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>nms_iou</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="maxBoxesNum">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox{
background:rgba(200, 200, 200,50);
color:white;
font-size: 14px;
font-family: &quot;Microsoft YaHei UI&quot;;
border-style: solid;
border-width: 1px;
border-color: rgba(200, 200, 200,100);
border-radius: 3px;}

QSpinBox::down-button{
background:rgba(200, 200, 200,0);
border-image: url(:/img/icon/箭头_列表展开.png);}
QDoubleSpinBox::down-button::hover{
background:rgba(200, 200, 200,100);
border-image: url(:/img/icon/箭头_列表展开.png);}

QSpinBox::up-button{
background:rgba(200, 200, 200,0);
border-image: url(:/img/icon/箭头_列表收起.png);}
QSpinBox::up-button::hover{
background:rgba(200, 200, 200,100);
border-image: url(:/img/icon/箭头_列表收起.png);}
</string>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="modelPathLabel">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>权重文件目录：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="nms_iouNum">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox{
background:rgba(200, 200, 200,50);
color:white;
font-size: 14px;
font-family: &quot;Microsoft YaHei UI&quot;;
border-style: solid;
border-width: 1px;
border-color: rgba(200, 200, 200,100);
border-radius: 3px;}

QDoubleSpinBox::down-button{
background:rgba(200, 200, 200,0);
border-image: url(:/img/icon/箭头_列表展开.png);}
QDoubleSpinBox::down-button::hover{
background:rgba(200, 200, 200,100);
border-image: url(:/img/icon/箭头_列表展开.png);}

QDoubleSpinBox::up-button{
background:rgba(200, 200, 200,0);
border-image: url(:/img/icon/箭头_列表收起.png);}
QDoubleSpinBox::up-button::hover{
background:rgba(200, 200, 200,100);
border-image: url(:/img/icon/箭头_列表收起.png);}
</string>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.050000000000000</double>
       </property>
       <property name="maximum">
        <double>0.950000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.300000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="modelPathEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="letterboxImageCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="detecAnsPathLabel">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>检测结果目录：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="detecAnsPathEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="detecAnsPathButton">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>file</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{font-family: &quot;Microsoft YaHei&quot;;
font-size: 14px;
font-weight: bold;
color:white;
text-align: center center;
padding-left: 5px;
padding-right: 5px;
padding-top: 4px;
padding-bottom: 4px;
border-style: solid;
border-width: 0px;
border-color: rgba(255, 255, 255, 255);
border-radius: 3px;
background-color: rgba(200, 200, 200,0);}

QPushButton:focus{outline: none;}

QPushButton::pressed{font-family: &quot;Microsoft YaHei&quot;;
                     font-size: 14px;
                     font-weight: bold;
                     color:rgb(200,200,200);
                     text-align: center center;
                     padding-left: 5px;
                     padding-right: 5px;
                     padding-top: 4px;
                     padding-bottom: 4px;
                     border-style: solid;
                     border-width: 0px;
                     border-color: rgba(255, 255, 255, 255);
                     border-radius: 3px;
                     background-color:  #bf513b;}

QPushButton::disabled{font-family: &quot;Microsoft YaHei&quot;;
                     font-size: 14px;
                     font-weight: bold;
                     color:rgb(200,200,200);
                     text-align: center center;
                     padding-left: 5px;
                     padding-right: 5px;
                     padding-top: 4px;
                     padding-bottom: 4px;
                     border-style: solid;
                     border-width: 0px;
                     border-color: rgba(255, 255, 255, 255);
                     border-radius: 3px;
                     background-color:  #bf513b;}
QPushButton::hover {
border-style: solid;
border-width: 0px;
border-radius: 0px;
background-color: rgba(48,148,243,80);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../apprcc.qrc">
         <normaloff>:/img/icon/打开.png</normaloff>:/img/icon/打开.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../apprcc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
